lib/web/desktop.go

```diff
@@ -222,7 +224,8 @@ func (h *Handler) createDesktopConnection(
 	if err != nil {
 		return sendTDPError(err)
 	}
-	for _, msg := range h.withheldMessages {
+	for _, msg := range withheld {
```

Comment:
Can we clear or otherwise nil-out the slice after we've sent the withheld messages so we don't have to hold them in memory for the duration of the session?

Reply:
c3c804eeff8240fbea0e05a2902b468572cc1358